### DISCUSSION TITLE
Algorithm Identifier displayed by asn1parse for Kyber variants

### DISCUSSION THREAD

**Discussion by Hussain1811** [openssl/openssl#24523] [Q&A] [answered]
`asn1parse` lists the algorithm identifiers as `kyber1024 `and `kyber768`. Should it be changed to display `mlkem1024` and `mlkem768` respectively?

**Comment by t8m** [comment]
Do MLKEM algorithms have the same OIDs as the respective KYBER algorithms?

**Reply by t-j-h** [reply, marked as answer]
They shouldn't as they are not actually identical

**Reply by Hussain1811** [reply]
Thanks @t-j-h and @t8m for the clarification
